be/src/olap/data_dir.cpp

```diff
@@ -804,4 +805,41 @@ Status DataDir::move_to_trash(const std::string& tablet_path) {
     return Status::OK();
 }
 
+void DataDir::perform_remote_gc() {
```

Comment:
DataDir::perform_path_gc_by_tablet() is used to delete unused file. Maybe rowset has remote file.